### PR TITLE
fix-dependencies

### PR DESCRIPTION
**Pull request by tbse**
setting dependencies to official npm repo

### CODE REVIEW

**Comment by moonglum** [comment]
@tbse Thanks for your contribution 🙇

File: test/test_basic/faucet.config.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ module.exports = {
 	},
 
 	plugins: {
-		"nunjucks": {
+		nunjucks: {
```
Comment (FND) [inline test/test_basic/faucet.config.js]: delete this repo

Comment (moonglum) [reply test/test_basic/faucet.config.js]: 😘

Comment (moonglum) [reply test/test_basic/faucet.config.js]: Dr. Evil airquoting "nunjucks"

Comment (FND) [reply test/test_basic/faucet.config.js]: 🍪

**Pull request merged**